docs/source/1.0/spec/aws/aws-query-protocol.rst

```diff
@@ -95,6 +95,27 @@ that affect serialization:
 
 .. |quoted shape name| replace:: ``awsQuery``
 .. |name resolution text| replace:: The :ref:`xmlName-trait` can be used to serialize a property using a custom name
+.. |query collection text| replace::
```

Comment:
Do you think the shared templates are still helping?

Reply:
Or maybe we can make them a little more granular?

Reply:
These big blocks aren't just copied into the template, they're repeated later on in the doc where there's a long-form examples.